code/modules/customitems/item_defines.dm

```diff
@@ -236,5 +236,25 @@
 	icon_state = "noble_boot"
 	_color = "noble_boot"
 	item_state = "noble_boot"
+	
+/////Arachno-Man Costume set //the flagbearer: Willow Walker
+/obj/item/clothing/under/fluff/arachno_suit // Custom Jumpsuit
+	name = "Arachno-Man costume"
+	desc = "It's what an evil genius would design if he switched brains with the Amazing Arachno-Man. Actually, he'd probably add weird tentacles that come out the back, too."
+	icon = 'icons/obj/clothing/uniforms.dmi'
+	icon_state = "superior_suit"
+	item_state = "superior_suit"
```

Comment:
You're going to need to define the _color var as "superior_suit" as well, otherwise the sprite won't appear on the person when worn, and it'll just say "jsuit" at the top right of the person wearing it.